third_party/nvfuser/csrc/scheduler/normalization.cpp

```diff
@@ -1828,16 +1828,26 @@ void schedulePersistentKernelInnerOuter(
   // directly from output tv using parallelizeAllLike. must propagate seperaely
   // for different tvs as outer reductions are transformed seperately.
   if (rparams.vectorization_factor_outer > 1) {
+    auto findVectorizedOutputOf = [&](TensorView* tv) {
+      TensorView* ref_tv = nullptr;
+      for (auto output_tv : ir_utils::outputTvsOf(tv)) {
```

Comment:
Is there any guarantee that any of the consumers of `cached_gmem_reload` is a fusion output?

Reply:
if `cached_gmem_reload` is not used(directly or indirectly) by the output, it shouldn't exist in the fusion. so we can guarantee at least one of its consumer (direct or indirect) is a fusion output.

Reply:
Sorry, I was confused with `consumerTvsOf`. Yes, `outputTvsOf` grabs all outputs, directly and indirectly, so this should be fine.